tensorflow/core/ops/nn_ops.cc

```diff
@@ -144,6 +144,45 @@ output: 4-D.  Gradients w.r.t. the input of `avg_pool`.
 
 // --------------------------------------------------------------------------
 
+#ifdef INTEL_MKL
+
+REGISTER_OP("MklAvgPool")
```

Comment:
Please add documentation for each input/output/attr. See docstring for other ops in this file. Documentation in these sections is important. Thanks.